test/library/draft/arrow/ParquetDecl.chpl

```diff
@@ -0,0 +1,40 @@
+module ParquetDecl {
+    require "parquet-glib/parquet-glib.h", "-lparquet-glib";
+    require "ArrayDecl.chpl", "ArrowRecordDecl.chpl";
+    private use ArrayDecl, ArrowRecordDecl;
+    use SysCTypes, CPtr;
+
+
+    // Types used in parquet-arrow.c
+    extern type GParquetArrowFileReader = c_void_ptr;
+    
+    extern type GParquetArrowFileWriter = c_void_ptr;
+    
+    extern type GParquetWriterProperties = c_void_ptr;
+
+    extern type GArrowTable = c_void_ptr;
+
+    extern type gchar = c_char;
```

Comment:
I had to add an `extern type guint64` declaration here. Am I missing something, or should that be in here?

Reply:
You're not missing anything, that should be in there. I didn't come across it when I pushed this because the `writeTableToParquetFile` function is still being worked on and I wasn't testing it. I've added that in there since and it will be there in the next push.

Reply:
Should be resolved now